packages/api/package.json

```diff
@@ -15,6 +15,7 @@
     "build": "WEBPACK_CLI_FORCE_LOAD_ESM_CONFIG=true webpack",
     "test": "npm-run-all -p -r mock:cluster mock:pgrest mock:backup test:e2e -s test:size",
     "test:e2e": "playwright-test \"test/**/*.spec.js\" --sw src/index.js -b webkit",
+    "test:e2e:debug": "playwright-test \"test/**/*.spec.js\" --debug --sw src/index.js -b webkit",
```

Comment:
```suggestion
    "test:e2e:debug": "npm run test:e2e -- --debug",
```

Reply:
Ack!

On a related note, is there any specific reason we have webkit, instead of chromium here?
I find the dev experience in chromium a bit nicer, is it ok to change it?

Reply:
we can't 😢  there is no support for chromium with the multipart upload [context](https://github.com/web3-storage/web3.storage/pull/110#issue-948841130)

Reply:
Thanks for the context! I thought you had a good reason for that :)